src/Controller/EnMarche/AssessorManagerController.php

```diff
@@ -0,0 +1,237 @@
+            return new Response();
+        }
+
+        return $this->render('assessor_manager/_requests_list.html.twig', [
+            'requests' => $requests,
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/demande/{id}",
+     *     requirements={"id": "\d+"},
+     *     name="app_assessor_manager_request",
+     *     methods={"GET"}
+     * )
+     * @Security("is_granted('MANAGE', assessorRequest)")
+     */
+    public function assessorRequestAction(
+        AssessorRequest $assessorRequest,
+        VotePlaceRepository $votePlaceRepository
+    ): Response {
+        return $this->render('assessor_manager/request.html.twig', [
+            'request' => $assessorRequest,
+            'matchingVotePlaces' => $votePlaceRepository->findMatchingVotePlaces($assessorRequest),
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/demande/{id}/associer/{votePlaceId}",
+     *     requirements={"id": "\d+"},
+     *     name="app_assessor_manager_request_associate",
+     *     methods={"GET", "POST"}
+     * )
+     * @ParamConverter("votePlace", class="AppBundle\Entity\VotePlace", options={"id": "votePlaceId"})
+     * @Security("is_granted('MANAGE', assessorRequest)")
+     */
+    public function assessorRequestAssociateAction(
+        Request $request,
+        VotePlaceRepository $votePlaceRepository,
+        AssessorManager $manager,
+        AssessorRequest $assessorRequest,
+        VotePlace $votePlace
+    ): Response {
+        if (!$assessorRequest->isEnabled() || !\in_array($votePlace, $votePlaceRepository->findMatchingVotePlaces($assessorRequest))) {
+            throw $this->createNotFoundException('No vote place for this request.');
+        }
+
+        $form = $this->createForm(ConfirmActionType::class)
+            ->handleRequest($request)
+        ;
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            if ($form->get('allow')->isClicked()) {
+                $manager->processAssessorRequest($assessorRequest, $votePlace);
+                $this->addFlash('info', 'assessor.associate.success');
+            }
+
+            return $this->redirectToRoute('app_assessor_manager_request', ['id' => $assessorRequest->getId()]);
+        }
+
+        return $this->render('assessor_manager/associate.html.twig', [
+            'form' => $form->createView(),
+            'request' => $assessorRequest,
+            'votePlace' => $votePlace,
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/demande/{id}/desassocier",
+     *     requirements={"id": "\d+"},
+     *     name="app_assessor_manager_request_deassociate",
+     *     methods={"GET", "POST"}
+     * )
+     * @Security("is_granted('MANAGE', assessorRequest)")
+     */
+    public function assessorRequestDessociateAction(
+        Request $request,
+        AssessorRequest $assessorRequest,
+        AssessorManager $manager
+    ): Response {
+        if (!$assessorRequest->getVoteCity()) {
```

Comment:
hm, maybe you want check votePlace instead of voteCity?

Reply:
:see_no_evil: